README.md

```diff
@@ -33,7 +33,7 @@
 ```
 $ git clone https://github.com/LalatenduMohanty/centos-live-iso.git
 $ cd centos-live-iso
-$ livecd-creator --config ./centos-7-minimal.ks --fslabel live-centos
```

Comment:
This line is used more than once in the document. Needs to be updated everywhere